models/pull.go

```diff
@@ -171,6 +173,39 @@ func (pr *PullRequest) loadProtectedBranch(e Engine) (err error) {
 	return
 }
 
+func (pr *PullRequest) requiredApprovals() int64 {
+	if pr.ProtectedBranch == nil {
+		if err := pr.loadProtectedBranch(x); err != nil {
+			log.Error("Error loading ProtectedBranch", err)
+		}
+	}
+	if pr.ProtectedBranch != nil {
+		return pr.ProtectedBranch.RequiredApprovals
+	}
+	return 0
+}
+
+// RequiresApproval returns true if this pull request requires approval, otherwise returns false
+func (pr *PullRequest) RequiresApproval() bool {
+	return pr.requiredApprovals() > 0
+}
+
+// GetReviewLabel returns the formatted review label with counter for the review of this pull request
+func (pr *PullRequest) GetReviewLabel(lang string) string {
+	if pr.RequiresApproval() {
+		rejections := pr.ProtectedBranch.GetRejectedReviewsCount(pr)
+		if pr.ProtectedBranch.GetRejectedReviewsCount(pr) > 0 {
+			return i18n.Tr(lang, "repo.pulls.review_rejected", rejections)
```

Comment:
translation messages ... shold not be use in models ... so if you need this should look at /router.. ors where you could add the function or services/pull/ ..?
@lunny where do you think should go the stuff witch doesnt touch db ...

Reply:
We should give some status on this method and put the language message on tmpl files.